server/src/analyzer/service.ts

```diff
@@ -48,6 +47,8 @@ export interface AnalysisResults {
 
 export type AnalysisCompleteCallback = (results: AnalysisResults) => void;
 
+const _configFileNames = ['pyrightconfig.json', 'mspython.json'];
```

Comment:
Do you think this should be "mspython.json" or "mspythonconfig.json"? I guess either one works. The former is more terse but also potentially a bit more ambiguous. I guess I don't have a strong opinion.

Surveying other similar config files, it looks like there's no consistent standard.
TypeScript: "tsconfig.json"
TSLint: "tslint.json"
Webpack: "webpack.config.json"
NPM: ".npmrc"
ESLint: ".eslintrc"